packages/code/src/Code.tsx

```diff
@@ -49,7 +53,26 @@ const copyStyle = css`
 `;
 
 const singleLineCopyStyle = css`
-  padding: 10px;
+  height: 36px;
+  padding-top: 0;
+  justify-content: center;
+`;
+
+const singleLineWrapperStyle = css`
+  align-items: center;
+  display: flex;
+
+  ${mq({
```

Comment:
Can we combine this and the below in a single media query

Reply:
Didn't know you could do this with facepaint 😮 Done!

Reply:
👍